.binder/environment.yml

```diff
@@ -0,0 +1,15 @@
+name: gdutils
```

Comment:
@kerfoot I reduced your env file to a bare minimum and created a lock file so binder starts faster. My next PR will be to package gdutils but you may want to save your current environment file until we do so.